examples/fftPlotAction.py

```diff
@@ -105,8 +105,8 @@ def fftAllCurves(self, checked=False):
 
         self.plot.clearCurves()
 
-        for i, curve in enumerate(allCurves):
-            x, y, legend, info, params = curve
+        for _i, curve in enumerate(allCurves):
```

Comment:
Probably a remnant from a past version. I agree that `i` and `enumerate` are no longer necessary.